docs/verify/simulate-with-qiskit-primitives.ipynb

```diff
@@ -0,0 +1,555 @@
+    "\n",
+    "      The primitives V1 result stores an array of values that can be accessed through the attribute\n",
+    "      [`EstimatorResult.values`](../api/qiskit/qiskit.primitives.EstimatorResult#values), where the`i`<sup>th</sup>\n",
+    "      element is the expectation value corresponding to the `i`<sup>th</sup> circuit and `i`<sup>th</sup> observable.\n",
+    "      To see the values for the first (and in this case, only) circuit evaluation, we must access the first item of the array.\n",
+    "\n",
+    "```python\n",
+    "print(f\" > Expectation value: {result.values[0]}\")\n",
+    "print(f\" > Metadata: {result.metadata}\")\n",
+    "```\n",
+    "\n",
+    "```text\n",
+    " > Expectation value: 3.999999999999999\n",
+    " > Metadata: [{}, {}, {}]\n",
+    "```\n",
+    "  </TabItem>\n",
+    "</Tabs>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set Estimator run options\n",
+    "\n",
+    "By default, the reference Estimator performs an exact statevector calculation based on the\n",
+    "[`quantum_info.Statevector`](../api/qiskit/qiskit.quantum_info.Statevector) class.\n",
+    "However, this can be modified to introduce the effect of the sampling overhead (also known as \"shot noise\").\n",
+    "\n",
+    "With the introduction of the new V2 interface, the sampling overhead in the Estimator is now more concretely defined.\n",
+    "The new interface accepts a `precision` argument that expresses the error bars that the\n",
+    "primitive implementation should target for expectation values estimates,\n",
+    "instead of the number of `shots` used in the V1 interface.\n",
+    "\n",
+    "<Tabs>\n",
+    "  <TabItem value=\"V2\" label=\"StatevectorEstimator (V2)\">\n",
+    "\n",
+    "      The sampling overhead is now called `precision`, and is defined exclusively at the `.run()`\n",
+    "      method level. This allows for a more finely-grained tuning of the option all the way to the PUB level.\n",
+    "\n",
+    "    ```python\n",
+    "    # Estimate expectation values for two PUBs, both with 0.05 precision.\n",
+    "    estimator_v2.run([(circuit1, obs_array1), (circuit2, obs_array_2)], precision=0.05)\n",
+    "    ```\n",
+    "\n",
+    "  </TabItem>\n",
+    "\n",
+    "  <TabItem value=\"V1\" label=\"Estimator (V1)\">\n",
+    "\n",
+    "      The V1 interface specifies shots in the following ways:\n",
+    "\n",
+    "        - Setting keyword arguments in the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method.\n",
+    "        - Modifying the [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) options.\n",
+    "\n",
+    "    ```python\n",
+    "    job = estimator.run(qc, observable, shots=2048, seed=123)\n",
+    "    result = job.result()\n",
+    "    print(result)\n",
+    "    ```\n",
+    "  </TabItem>\n",
+    "</Tabs>"
+   ]
+  },
```

Comment:
Do we need output displayed for these tabs?

Reply:
Yes. My take would be: convert the python snippets to code cells, and these will automatically display whatever is in the `print` (and maybe add prints where needed).